15-morning-sun-star/api_definitions/rest/user_registry.yaml

```diff
@@ -0,0 +1,257 @@
+openapi: 3.1.0
+
+info:
+  title: User Registry API
+  version: 0.1.0
+  description: REST API for managing the GHGA user registry
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/
+  contact: 
+    url: https://www.ghga.de/about-us/contact
+
+servers:
+  - url: https://api.server.test/v1
+
+tags: 
+  - name: users 
+    description: User Data
+
+paths:
+
+  /users:
+    post:
+      operationId: post_user
+      summary: Register a user
+      description: Endpoint used to register a new user
+      tags:
+        - users
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                id:
+                  title: External ID
+                  type: string
+                issuer:
+                  title: Issuer
+                  type: string
+                name:
```

Comment:
Maybe it is better to distinguish between first name and last name.
Moreover, for sending messages, it would be nice to also collect the gender (and optionally the academic title).

Reply:
We need to clarify first if/how we can get the name components via OIDC. Officially there is only the combined name attribute. Added the academic title.